src/main.c

```diff
@@ -730,19 +733,66 @@ static h2o_iovec_t *build_http2_origin_frame(h2o_configurator_command_t *cmd, yo
     return http2_origin_frame;
 }
 
+static ptls_cipher_suite_t **parse_tls13_ciphers(h2o_configurator_command_t *cmd, yoml_t *node, const char *tls13_ciphers)
```

Comment:
I do not think we need to pass `tls13_ciphers` as a separate argument, as it can be obtained as `node->data.scalar`?